audio/orca/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- vim:fenc=utf-8:ft=tcl:et:sw=2:ts=2:sts=2
+
+PortSystem              1.0
+PortGroup               makefile 1.0
+
+name                    orca
+version                 20220224
+revision                0
+
+categories              audio lang
+license                 MIT
+maintainers             {@aeiouaeiouaeiouaeiouaeiouaeiou outlook.com:aeioudev} openmaintainer
+
+description             C implementation of the ORCΛ language \
+                        and terminal livecoding environment.
+long_description        Orca is a two-dimensional esoteric programming \
+                        language in which every letter of the alphabet \
+                        is an operator, where lowercase letters operate \
+                        on bang, uppercase letters operate each frame. The \
+                        application is not a synthesiser, but a flexible \
+                        livecoding environment capable of sending MIDI, \
+                        OSC & UDP to your audio interface, like Ableton, \
+                        Renoise, VCV Rack or SuperCollider.
+
+fetch.type              git
+git.url                 https://github.com/hundredrabbits/Orca-c
+git.branch              d027a414d5ff257b52c613284673933077a53cfa
+
+homepage                https://100r.co/site/orca.html
+
+depends_lib             port:ncurses \
+                        port:portmidi
+
+patchfiles              patch-tool.diff
+
+post-patch {
+        reinplace "s|@PREFIX@|${prefix}|g" ${worksrcpath}/tool
+    }
+
+build.target            release
+
+destroot {
+        xinstall -m 0755 ${build.dir}/build/orca ${destroot}${prefix}/bin
+    }
```

Comment:
the intendation of the `post-patch` and `destroot` phases can have 4 spaces removed